xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -1665,6 +1665,7 @@ CDemuxStream* CDVDDemuxFFmpeg::AddStream(int streamIdx)
 
         int size = 0;
         uint8_t* side_data = nullptr;
+        st->hdr_type = StreamHdrType::HDR_TYPE_NONE;
```

Comment:
Just my 2 cents on the matter, I am not really into this stuff but I've come recently across similar situations - e.g. when detecting and storing state for optical drives/trays. I'd say `HDR_TYPE_NONE` has a predefined semantics and a specific meaning -> it means the stream under processing has no HDR flag. Wouldn't it make more sense to have a `HDR_TYPE_UNKNOWN` generic flag that could be used on initialization or if the HDR type could not be determined?
We seem to follow this "pattern" in other parts of the code and at this point you still don't know if it's an HDR stream or not. Seems wrong to flag it right away as `HDR_TYPE_NONE`. Later on you might depend on this predefined meaning to take some action or do further processing on the stream (or other components)